src/Broadway/Replaying/Replayer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Broadway\Replaying;
+
+use Broadway\EventStore\EventVisitor;
+use Broadway\EventStore\Management\Criteria;
+use Broadway\EventStore\Management\EventStoreManagement;
+
+abstract class Replayer
```

Comment:
Why is this an abstract class? There's no interface or abstract methods to implement?